DolbyIO/Source/Private/DolbyIOVideoSink.cpp

```diff
@@ -109,10 +111,12 @@ namespace DolbyIO
 			    static_cast<uint8*>(CVPixelBufferGetBaseAddressOfPlane(PixelBuffer, 0)),
 			    CVPixelBufferGetBytesPerRowOfPlane(PixelBuffer, 0),
 			    static_cast<uint8*>(CVPixelBufferGetBaseAddressOfPlane(PixelBuffer, 1)),
-			    CVPixelBufferGetBytesPerRowOfPlane(PixelBuffer, 1), Buffer.GetData(), VideoFrame.width() * Stride,
+			    CVPixelBufferGetBytesPerRowOfPlane(PixelBuffer, 1), Buffer, VideoFrame.width() * Stride,
 			    VideoFrame.width(), VideoFrame.height());
 		}
 #endif
+		Texture->PLATFORM_DATA->Mips[0].BulkData.Unlock();
```

Comment:
Is it possible to not reach this line by having an exception thrown above it?

Reply:
good point - even if not now it may start happening in the future.